SPECIFICATION.md

```diff
@@ -27,7 +27,9 @@ enough RAM, etc.), customers can rely on the following:
      `<= 5 MiB`. The memory overhead is dependent on the **workload** (e.g. a
```

Comment:
This doc change should be mentioned in changelog

Reply:
It is not clear to me if you are referring to the change below, added by this PR, or to this:
`Firecracker's virtual machine manager threads have a memory overhead
   - `Firecracker's virtual machine manager threads have a memory overhead
     <= 5 MiB. The memory overhead is dependent on the **workload**`

Reply:
We should add in changelog that MMDS documentation was improved and now there are two different sections MMDS Design and MMDS user guide

Reply:
I don't see the changelog modification.